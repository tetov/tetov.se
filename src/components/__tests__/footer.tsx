import React from "react";
import renderer from "react-test-renderer";
import Footer from "src/components/footer";

describe("Footer", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Footer>Test</Footer>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
