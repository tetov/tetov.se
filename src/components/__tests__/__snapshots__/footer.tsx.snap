// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="pt-8 pb-16 items-center"
>
  Test
</footer>
`;
